frontend/src/reviews/ReviewDecisionDisplay.tsx

```diff
@@ -69,16 +112,41 @@ export default function ReviewDecisionDisplay({ response, modelId }: ReviewDecis
                 <Typography variant='caption'>as {getRoleDisplay(response.role, modelRoles)}</Typography>
               )}
             </Stack>
-            <Typography fontWeight='bold'>{formatDateString(response.createdAt)}</Typography>
+            <Stack direction='row' alignItems='center' spacing={1}>
+              <Typography fontWeight='bold'>{formatDateString(response.createdAt)}</Typography>
+              <IconButton onClick={(event) => setAnchorEl(event.currentTarget)}>
+                <MenuIcon />
+              </IconButton>
+            </Stack>
           </Stack>
-          {response.comment && (
-            <div>
-              <Divider sx={{ mt: 1, mb: 2 }} />
+          <Divider sx={{ mt: 1, mb: 2 }} />
+          {response.comment && !isEditMode && (
```

Comment:
The logic between here and line 142 looks exactly the same as in `ReviewCommentDisplay`. We should pull this out into a component rather than duplicating code.